src/components/pages/coach/GenerateReport/index.jsx

```diff
@@ -325,12 +350,11 @@ class GenerateReport extends React.Component{
     }
 
     getLink = () => {
-        console.log("User ID",this.props.location.state.user.id);
         this.props.client.query({
             query: GET_LINK,
             variables: { 
                 "token": localStorage.getItem("fprint"),
```

Comment:
"fprint" doesn't seem to be the best naming for an JWT in local storage.
I would prefer to name it anonymousJWT or something along those lines,
cause it's the JWT of the anonymous user.

Reply:
I have to agree on that one. They are a remenant of darker times. 🌋